src/common/actions/science/technologyactions.cpp

```diff
@@ -23,34 +23,34 @@
 
 namespace cqsp::common::actions {
 
-void ResearchTech(Universe& universe, entt::entity civilization, entt::entity tech) {
+void ResearchTech(Node civilization, Node tech) {
```

Comment:
I think node should probably be a reference? Not sure if that's more appropriate